src/main/java/com/google/api/codegen/php/PhpDiscoveryContext.java

```diff
@@ -98,14 +101,14 @@ public void resetState(PhpSnippetSet<?> phpSnippetSet, PhpContextCommon phpCommo
    * Returns the method names used in the PHP client code.
    */
   public String getMethodName(Method method) {
-    String name = super.getMethodName(method);
-    if (RENAMED_METHODS.contains(name)) {
+    StringBuilder builder = new StringBuilder(super.getMethodName(method));
+    if (RENAMED_METHODS.contains(builder.toString())) {
```

Comment:
It feels strange to call toString here.
`contains(super.getMethodName(method))` ?